include/exotic/cester.h

```diff
@@ -3194,10 +3195,12 @@ static __CESTER_INLINE__ void cester_run_test(TestInstance *test_instance, TestC
         last_status = superTestInstance.current_execution_status;
 #ifdef __STDC_VERSION__
 #ifndef __cplusplus
+#ifndef CASTER_NO_SUBPROCESS

```

Comment:
The macro should start with CESTER_, not CASTER_

Reply:
Done.